core/templates/dev/head/forms/formBuilder.js

```diff
@@ -723,6 +723,7 @@ oppia.config(['$provide', function($provide) {
       var _openCustomizationModal = function(
           customizationArgSpecs, attrsCustomizationArgsDict, onSubmitCallback,
           onDismissCallback, refocusFn) {
+        $document[0].execCommand('enableObjectResizing', false, false);
```

Comment:
Angular requires the [0] for execCommand (why I originally had issues with using $document).